trampoline/src/main/java/org/ernest/applications/trampoline/services/EncryptService.java

```diff
@@ -0,0 +1,41 @@
+package org.ernest.applications.trampoline.services;
```

Comment:
Maybe we should just keep this simple (less code) and just initialize a `StandardPBEStringEncryptor` bean?

Reply:
Thought about it but may be it's better to have an entity responsible for that task and encryptor used bing encapsulated there since when you want to use it, you will autowired an EncryptService rather than a concrete Encryptor implementation. Moreover, it will help whoever will want to change/refactor the encryptor method. Although both are fine, I feel solution applied is cleanner and more mantanaible in a long term. I convinced you?

Reply:
Yes, sounds good.